ports/esp32s2/common-hal/wifi/Radio.c

```diff
@@ -149,13 +150,66 @@ wifi_radio_error_t common_hal_wifi_radio_connect(wifi_radio_obj_t *self, uint8_t
     return WIFI_RADIO_ERROR_NONE;
 }
 
+mp_obj_t common_hal_wifi_radio_get_ap_info(wifi_radio_obj_t *self) {
+    if (!esp_netif_is_netif_up(self->netif)) {
+        return mp_const_none;
+    }
+
+    // Make sure the interface is in STA mode
+    if (self->sta_mode){
```

Comment:
```suggestion
    if (!self->sta_mode){
```

Reply:
You're right!  But this made me wonder why my tests worked at all.  I don't see any reference to anything actually setting ```self->sta_mode``` to True at any point...I'm just double checking to make sure I didn't goof up and remove that somewhere else or if it was just forgotten during ```start_station```.